RELEASE/scripts/autoscend/paths/bees_hate_you.ash

```diff
@@ -3,7 +3,7 @@ script "bees_hate_you.ash"
 
 void bhy_initializeSettings()
 {
-	if(auto_my_path() == "Bees hate You")
+	if(auto_my_path() == "Bees Hate You")
```

Comment:
I suggest making a `in_bhy()` function that does this at the top of this file and using it instead as `LM_bhy()` also has the same issue and it'll make sure it doesn't come up again.

Reply:
Should I refactor to use in_bhy() everywhere or just in bees_hate_you.ash?

Reply:
everywhere